### PR TITLE
Zeebe 8.5.0 breaks serialization of timestamp values in management API

### PR DESCRIPTION
**Pull request by oleschoenburg**
## Description

Documents that Zeebe 8.5.0 is affected by https://github.com/camunda/zeebe/issues/17347.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ description: "Important announcements including deprecation & removal notices"
 
 ## Camunda 8.5
 
+Release date: 9th of April 2024
```
Comment (megglos) [inline docs/reference/announcements.md:9]: thanks for fixing this! ❤️

File: docs/reference/announcements.md [hunk 2]
```diff
@@ -55,6 +55,13 @@ Note that the actual values shown in this screenshot don't correspond to any act
 As of the 8.5 release, the Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
 For a migration guide, see the [Web Modeler API documentation](/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
 
+### Zeebe 8.5.0 breaks serialization of timestamp values in management API
+
+Zeebe 8.5.0 was released with [a new bug](https://github.com/camunda/zeebe/issues/17347) that breaks serialization of timestamp values in management APIs such as the backup and cluster scaling API.
```
Comment (megglos) [inline docs/reference/announcements.md:60]: 🔧  let's link the APIs for context
```suggestion
Zeebe 8.5.0 was released with [a new bug](https://github.com/camunda/zeebe/issues/17347) that breaks serialization of timestamp values in management APIs such as the [backup](/self-managed/operational-guides/backup-restore/backup-and-restore.md) and [cluster scaling API](/self-managed/zeebe-deployment/operations/cluster-scaling.md).
```

File: docs/reference/announcements.md [hunk 3]
```diff
@@ -55,6 +55,13 @@ Note that the actual values shown in this screenshot don't correspond to any act
 As of the 8.5 release, the Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
 For a migration guide, see the [Web Modeler API documentation](/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
 
+### Zeebe 8.5.0 breaks serialization of timestamp values in management API
```
Comment (megglos) [inline docs/reference/announcements.md:58]:
```suggestion
### Zeebe 8.5.0 Self-Managed breaks serialization of timestamp values in management API
```

File: versioned_docs/version-8.5/reference/announcements.md [hunk 4]
```diff
@@ -55,6 +55,13 @@ Note that the actual values shown in this screenshot don't correspond to any act
 As of the 8.5 release, the Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
 For a migration guide, see the [Web Modeler API documentation](/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
 
+### Zeebe 8.5.0 breaks serialization of timestamp values in management API
+
+Zeebe 8.5.0 was released with [a new bug](https://github.com/camunda/zeebe/issues/17347) that breaks serialization of timestamp values in management APIs such as the backup and cluster scaling API.
```
Comment (megglos) [inline versioned_docs/version-8.5/reference/announcements.md:60]: 🔧  let's link the APIs for context
```suggestion
Zeebe 8.5.0 was released with [a new bug](https://github.com/camunda/zeebe/issues/17347) that breaks serialization of timestamp values in management APIs such as the [backup](/self-managed/operational-guides/backup-restore/backup-and-restore.md) and [cluster scaling API](/self-managed/zeebe-deployment/operations/cluster-scaling.md).
```

**Review by megglos — APPROVED** [review]
LGTM! Thanks for fixing the misplacement of the Release date and end of maintenance lines along the way!
Some suggestions to link the mentioned APIs and adding an indication this is only relevant for self-managed.

**Review by megglos — APPROVED** [review]
LGTM! Missed the SM addition to the versioned docs 🙈 pre-approved

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, but is this worth noting as an admonition across the related SM content as well, such as the linked docs you mentioned?

**Comment by oleschoenburg** [comment]
> but is this worth noting as an admonition across the related SM content as well, such as the linked docs you mentioned?

I'd say no because it's just a temporary bug that will be fixed with the next patch.

**Pull request merged**